packages/data-row-permissions/src/main.js

```diff
@@ -0,0 +1,67 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+
+import i18n from '@molgenis/molgenis-i18n-js'
+import { BootstrapVue } from 'bootstrap-vue'
+
+// Import Bootstrap an BootstrapVue CSS files (order is important)
+import 'bootstrap/dist/css/bootstrap.css'
```

Comment:
Use molgenis theme instead.

Reply:
Molgenis theme does work (have tested) It will overwrite